src/modules/partition/core/PartitionActions.cpp

```diff
@@ -144,6 +149,26 @@ doAutopartition( PartitionCoreModule* core, Device* dev, Choices::AutoPartitionO
         }
         core->createPartition( dev, efiPartition, KPM_PARTITION_FLAG_ESP );
         firstFreeSector = lastSector + 1;
+
+        if ( createHybridBootloaderLayout )
+        {
+            // TODO: Avoid hardcoding 8 MiB?
+            qint64 bios_part_sizeB = 8388608;
```

Comment:
537:    qint64 uefisys_part_sizeB = 300_MiB;

Reply:
Oh nice, didn't know that we had `_MiB` literals in Cala. I see them now. Will use this.